### PR TITLE
Changing custom repos options to match amigen8

### PR DESCRIPTION
**Pull request by oblandford-novetta**
Changes offered/proposed in this pull request:
- modifying the customrepos variables to mirror the amigenrepos variables used in amigen8.sh
- modified the devnode variable used to map the device for the volume to amigenbuilddev to mirror amigen8.sh

* New PR Alert to: @plus3it/spel

### CODE REVIEW

File: spel/minimal-linux.json [hunk 1]
```diff
@@ -587,4 +587,4 @@
         "vagrantcloud_token": "{{env `VAGRANTCLOUD_TOKEN`}}",
         "vagrantcloud_username": ""
     }
-}
+}
```
Comment (lorengordon) [inline spel/minimal-linux.json:590]: This is indicating a new line is missing (recommend installing the editorconfig plugin)

File: README.md [hunk 2]
```diff
@@ -267,8 +267,8 @@ defaults):
     "spel_amigen7storlay": "/:rootVol:4,swap:swapVol:2,/home:homeVol:1,/var:varVol:2,/var/log:logVol:2,/var/log/audit:auditVol:100%FREE",
     "spel_amiutilsource": "https://github.com/ferricoxide/Lx-GetAMI-Utils.git",
     "spel_awsclisource": "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip",
-    "spel_customreponame7": "",
-    "spel_customreporpm7": "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm",
+    "spel_amigen7reponames": "",
```
Comment (lorengordon) [inline README.md:270]: Keep these sorted

File: README.md [hunk 3]
```diff
@@ -296,11 +296,11 @@ defaults):
 | `spel_amigen7storlay`   | List of MOUNT:VOLNAME:VOLSIZE tuples to customize storage-layout  |
 | `spel_amiutilsource`    | URL to the git repository for the `Lx-GetAMI-Utils` project       |
 | `spel_awsclisource`     | URL to the site hosting the file `awscli-bundle.zip`              |
-| `spel_customreporpm7`   | URL to a custom release RPM containing base repos for EL7         |
-| `spel_customreponame7`  | Name(s) of the custom yum repos (* or comma-separated) for EL7    |
+| `spel_amigen7reposource`   | URL to a custom release RPM containing base repos for EL7         |
```
Comment (lorengordon) [inline README.md:299]: Keep these sorted, and re-align the `|` chars to keep it clean

File: build/pre_build.sh [hunk 4]
```diff
@@ -21,9 +21,9 @@ AWS_PROFILE="$SPEL_IDENTIFIER" ./packer validate \
   -var "spel_amiutilsource=$SPEL_AMIUTILSOURCE" \
   -var "spel_aws_cliv1_source=$SPEL_AWSCLIV1SOURCE" \
   -var "spel_aws_cliv2_source=$SPEL_AWSCLIV2SOURCE" \
-  -var "spel_customreporpm7=$SPEL_CUSTOMREPORPM7" \
-  -var "spel_customreponame7=$SPEL_CUSTOMREPONAME7" \
-  -var "spel_devnode=$SPEL_DEVNODE" \
+  -var "spel_amigen7reposource=$SPEL_AMIGENREPOSRC7" \
```
Comment (lorengordon) [inline build/pre_build.sh:24]: Also sorted...

File: spel/minimal-linux.json [hunk 5]
```diff
@@ -551,7 +551,7 @@
         "openstack_network_ids": "",
         "openstack_security_group_names": "",
         "openstack_source_image_name": "",
-        "root_volume_size": "20",
+        "root_volume_size": "30",
```
[the remark this answers is not in the document]
Comment (oblandford-novetta) [reply spel/minimal-linux.json:554]: this was a test that I should not have committed. It was looking into Long's question about increasing the size of the EBS volume before publishing the AMIs. This is the variable which handles it. I was just confirming. I will remove the change and if we want to modify it we will do it in a separate commit

**Comment by oblandford-novetta** [comment]
go codebuild go

**Review by lorengordon — APPROVED** [review]
nice work, thanks @oblandford-novetta! 🚢

**Pull request merged**